src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsTableHeaderUI.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1997, 2014, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1997, 2014, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 1997, 2022, Oracle and/or its affiliates. All rights reserved.
```
Nope, you don't add another year, you just modify the second year to the current one.

If the file has only one year, then you add the second one.